modules/hamronization/fargene/meta.yml

```diff
@@ -0,0 +1,62 @@
+name: "hamronization_fargene"
+description: Tool to convert and summarize fARGene outputs using the hAMRonization specification.
+keywords:
+  - amr
+  - antimicrobial resistance
+  - arg
+  - antimicrobial resistance genes
+  - reporting
+  - fARGene
+tools:
+  - hamronization:
+      description: "Tool to convert and summarize AMR gene detection outputs using the hAMRonization specification"
+      homepage: "https://github.com/pha4ge/hAMRonization/blob/master/README.md"
```

Comment:
I just noticed (also for other modules) that this points to the `README.md`s. Is that preferred or better point to the whole repo?

Reply:
Ah just the repo... will update that!